docs/src/turing.md

```diff
@@ -0,0 +1,100 @@
+# Using Pathfinder to initializing sampling with Turing
+
+If you have a _differentiable_ Turing model, then you can use Pathfinder to initialize its parameters.
```

Comment:
Just my two cents. _Differentiable_ Turing model means one that do not have for loops and integers? Like a model which would run in Zygote or ReverseDiff?

Maybe explain with a footnote?

Reply:
Thanks, yeah I wrote the text hastily. Should say the technical requirements are that all parameters are continuous and the log-density can be differentiated by an AD framework.

Reply:
Yes, that what I got the intuition. But a footnote would solve this all. Thanks!